### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by svillegasz**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+ "defaultSeverity": "error",
+ "extends": [
+   "tslint-config-airbnb"
+ ],
+ "rules": {
+   "trailing-comma": [true]
+ }
+}
```
Comment (aperdomob) [inline tslint.json]: EOF

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
-[![Build Status](https://travis-ci.org/svillegasz/psl-workshop-protractor.svg?branch=master)]
+[![Build Status](https://travis-ci.org/svillegasz/psl-workshop-api-testing-js.svg?branch=master)](https://travis-ci.org/svillegasz/psl-workshop-api-testing-js)
```
Comment (jhenaoz) [inline README.md:1]: 👍

File: package.json [hunk 3]
```diff
@@ -11,6 +11,7 @@
     "test:local": "npm run build && protractor dist/protractor/config.js",
     "test:headless": "npm run build && protractor dist/protractor/headless.config.js",
     "test": "npm run test:headless",
+    "lint": "tslint --type-check --project tsconfig.json protract/**/*.ts test/**/*.ts src/**/*.ts",
```
Comment (jhenaoz) [inline package.json]: TYPO:
cambiar `"lint": "tslint --type-check --project tsconfig.json protract/**/*.ts test/**/*.ts src/**/*.ts"`
por: `"lint": "tslint --type-check --project tsconfig.json protractor/**/*.ts test/**/*.ts src/**/*.ts"`

**Review by jhenaoz — DISMISSED** [review]
pusiste a escanear la carpeta protrac en lugar de protractor

**Review by aperdomob — APPROVED** [review]

**Comment by svillegasz** [comment]
@jhenaoz

**Pull request merged**